src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -256,11 +256,14 @@
         --network-plugin=azure will use an overlay network (non-VNET IPs) for pods in the cluster.
   - name: --network-policy
     type: string
-    short-summary: The Kubernetes network policy to use.
+    short-summary: Network Policy engine to use.
     long-summary: |
-        Using together with "azure" network plugin.
-        Specify "azure" for Azure network policy manager, "calico" for calico network policy controller, "cilium" for Azure CNI powered by Cilium.
-        Defaults to "" (network policy disabled).
+        Azure provides three Network Policy engines for enforcing network policies that can be used together with "azure" network plugin. The following values can be specified:
+          - "azure" for Azure Network Policy Manager,
+          - "cilium" for Azure CNI Overlay powered by Cilium,
+          - "calico" for open-source network and network security solution founded by Tigera,
+          - "none" to uninstall Azure Network Policy Manager or Calico.
```

Comment:
"none" means no network policy engine installed, not necessarily uninstall

Reply:
Right, thanks for the catch. Updated descriptions for "none" network policy in both create and update routes.